library/invoice.ts

```diff
@@ -118,6 +118,7 @@ async function handleCompletedAmpInvoice(
     contentTime: timestamp ?? null,
     userId: pubkey ? pubkey : null,
     comment: content ? content : null,
+    isNostr: paymentTypeCode === PaymentType.Zap ? true : false,
```

Comment:
nit - no need for ternary
```suggestion
    isNostr: paymentTypeCode === PaymentType.Zap,
```